src/models/modules/__init__.py

```diff
@@ -1,11 +1,11 @@
 """All the modules used in creation of Generator and Discriminator"""
-from .discriminator import Discriminator, FeedbackModule
 from .acm import ACM
 from .attention import ChannelWiseAttention, SpatialAttention
 from .cond_augment import CondAugmentation
 from .conv_utils import calc_out_conv, conv1d, conv2d
+from .discriminator import Discriminator, FeedbackModule
 from .downsample import down_sample
 from .generator import Generator
-from .image_encoder import ImageEncoder
+from .image_encoder import ImageEncoder, VGGEncoder
```

Comment:
naming of ImageEncoder is ambiguous now since we have at least two image encoders. what do you think about renaming ImageEncoder into something more specific like InceptionEncoder? Since we have VGGEncoder, I don't think it should be a problem

Reply:
updated it, please check and review